### PR TITLE
Removing SimplePolicy plugin

### LINKED ISSUE #109: "IsLowPriority"  need to be removed in SimplePolicy of NEO3
"IsLowPriority"  need to be removed in SimplePolicy since Transcation.get_IsLowPrioriy() has been removed.

image

### PR DESCRIPTION
**Pull request by vncoelho**
Closes #109 

@erikzhang, @shargon, @igormcoelho, I just left the `void ILogPlugin.Log(string source, LogLevel level, string message)`

I think that we should also rename this class from `SimplePolicy` -> `Loggers`, something like that.
Many people were used to have `SimplePolicy` as default and a requirement. In this sense, it will be good to change the name for avoiding confusion.

### CODE REVIEW

**Comment by shargon** [comment]
I think that we should use our own branch

File: SimplePolicy/SimplePolicyPlugin.cs [hunk 1]
```diff
@@ -1,83 +1,14 @@
...
-                .ThenBy(p => p.Hash)
-                .Take(Settings.Default.MaxTransactionsPerBlock - free.Length - 1)
-                .ToArray();
-
-            return non_free.Concat(free);
-        }
-
         void ILogPlugin.Log(string source, LogLevel level, string message)
         {
             if (source != nameof(ConsensusService)) return;
```
Comment (shargon) [inline SimplePolicy/SimplePolicyPlugin.cs]: this should be moved to ApplicationLogs

File: .travis.yml [hunk 2]
```diff
@@ -1,19 +0,0 @@
...
- - osx
-
-dist: bionic
-osx_image: xcode9.1
-
-mono: none
-dotnet: 2.1.300
-
-before_install:
-  - cd SimplePolicy.UnitTests
```
Comment (shargon) [inline .travis.yml]: We should have a travis, just for compilation, and in the future make test for plugins too, but i think that we need it

**Comment by vncoelho** [comment]
@shargon, I prefer to use neo branch, I think it is easy for contributions.

Nowadays, there are not many branches in the project. We just need to remember to clean it.
But if that becomes a requirement I do not mind in using my own repo. But nowadays I prefer using here, looks simpler to me and I do not need to give permissions for other devs to edit.

**Comment by shargon** [comment]
For me is worst, i use gitkraken and i have no rights with neo branch, in neo-sharp we worked with our own branch and i think that was a good practice

**Comment by erikzhang** [comment]
We can just remove the plugin and move logger to neo core.

**Comment by erikzhang** [comment]
We can just remove the SimplePolicy plugin and move the logger to ApplicationLogs.

File: .travis.yml (hunk 2 above)
Comment (vncoelho) [reply .travis.yml]: I think it is not needed it there are no tests.
Sometime ago I did a template for each plugin and it was rejected.

Comment (vncoelho) [reply .travis.yml]: But it looks strange to have the yellow simbol of travis forever. Maybe we can leave the file as you suggested.

Comment (erikzhang) [reply .travis.yml]: Let's leave the file.

File: SimplePolicy/SimplePolicyPlugin.cs (hunk 1 above)
Comment (erikzhang) [reply SimplePolicy/SimplePolicyPlugin.cs]: Maybe we should create a new plugin to categorize and save all the logs, not just the consensus logs. I think ApplicationLogs is not used to do these logging, it is only used to record the results of the Application execution.

File: .travis.yml (hunk 2 above)
Comment (vncoelho) [reply .travis.yml]: Re-added without scripts

Comment (erikzhang) [reply .travis.yml]: We can add `dotnet restore` to make it build automatically.

Comment (vncoelho) [reply .travis.yml]: Done

Comment (erikzhang) [reply .travis.yml]: Travis failed.

You can revert the changes to the `.travis.yml` and remove the line `dotnet test` only.

Comment (vncoelho) [reply .travis.yml]: It is ok, we need to remove the `before_install` as well.
I think that I forgot to clean the .sln

Comment (erikzhang) [reply .travis.yml]: You should clean `SimplePolicy.UnitTests` from .sln too. Why don't you use Visual Studio?

Comment (vncoelho) [reply .travis.yml]: AEHuaheuahea, until nowadays using `gedit` or editing here online.
I need to configure visual studio code. I will do that as soon as possible.

**Pull request merged**